frontend/coalesce/src/pages/CreateAnOpportunityPage.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+  <q-page class="q-px-xl q-mx-xl">
+    <div class="text-h2 text-weight-bold q-pt-xl">Create an Opportunity</div>
+    <div class="text-body1 q-py-lg">A little bit of info about creating a volunteering opportunity</div>
+
+    <div class="q-pb-lg">
+      <q-btn color="secondary" label="Submit Opportunity" />
+    </div>
+
+    <div class="text-overline">OPPORTUNITY DETAILS</div>
+    <q-card class="my-card">
+      <q-card-section>
+        <q-input v-model="text" label="Opportunity Title" stack-label :dense="dense" />
+        <q-input v-model="text" label="Description" stack-label :dense="dense" />
+        <q-input v-model="text" label="Location" stack-label :dense="dense" />
```

Comment:
The textbox inputs are using the same `text` model field so they overwrite each other. I think it will be necessary to give each one its own field in the components `data` object. (e.g. `opportunity_title`, `description`, etc).